mesh/mesh.cpp

```diff
@@ -777,13 +777,39 @@ const GeometricFactors* Mesh::GetGeometricFactors(const IntegrationRule& ir,
    return gf;
 }
 
+const FaceGeometricFactors* Mesh::GetFaceGeometricFactors(
```

Comment:
Again, adding new `face` features in `mesh` could be avoided if we can come with new files.

Reply:
I'm not sure we can easily get rid of this function since the `FaceGeometricFactors` are stored in the mesh similarly to the `GeometricFactors`.